openapi/components/schemas/QuoteReactivateOrder.yaml

```
@@ -0,0 +1,488 @@
+type: object
+title: QuoteReactivateOrder
+required:
+  - websiteId
+  - customerId
+  - items
+  - action
+  - orderId
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
+    type: string
+    readOnly: true
+    enum:
+      - subscription-order
+      - one-time-order
+  action:
+    description: |-
+      Action of the quote for churned/canceled order reactivation.
```

suggestion: for readability we should avoid using '/' 
```suggestion
      Action of the quote for churned or canceled order reactivation.
```